modules/apps/frontend-js/frontend-js-components-web/test/treeview/Treeview.js

```diff
@@ -276,4 +276,57 @@ describe('Treeview', () => {
 			);
 		});
 	});
+
+	describe('Treeview with filter prop', () => {
+		it('when a string is passed it compares it with the node names ignoring case', () => {
```

Comment:
I think this is a pet-peeve of @wincent's, but we're trying to push to [Start it() descriptions with a verb, not with "should"](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/testing/anatomy.md#start-it-descriptions-with-a-verb-not-with-should). Obviously a when is not a should, but it isn't a verb either 😁

Reply:
Yeah, makes a lot of sense, I think I messed up after adapting the description 😂 I'll update that, thanks!

Reply:
Done! :)